setup.py

```diff
@@ -360,7 +360,7 @@ def get_dynamic_setup_params():
     url='https://ansible.com/',
     project_urls={
         'Bug Tracker': 'https://github.com/ansible/ansible/issues',
-        'CI: Shippable': 'https://app.shippable.com/github/ansible/ansible',
+        'CI: Shippable': 'https://dev.azure.com/ansible/ansible/',
```

Comment:
Hrm, I need to remove `Shippable` from the key too.

Reply:
Oops, sorry missed that.